Updating to look like en_US.lang lattest format

efimov90 opened the pull request:
some string work:
sorting like en_EN, removing, updating some strings to new strings, partial translating

проведена работа со строками:
перемещение, замена неработающих строк на обновленные аналоги, удаление лишних строк, частичный перевод дополняющих строк.

efimov90 committed:
приведение к en_EN

проведена работа со строками:
перемещение, замена неработающих строк на обновленные аналоги, удаление лишних строк, частичный перевод дополняющих строк.

liach commented on assets/railcraft/lang/ru_RU.lang:
```diff
@@ -394,44 +473,30 @@ railcraft.gui.help=Помощь
+gui.railcraft.manipulator.buffer=Буфер
+gui.railcraft.manipulator.redstone.complete.name=Завершено
+gui.railcraft.manipulator.redstone.complete.tips=Продолжать до конца операции
+gui.railcraft.manipulator.redstone.immediate.name=Immediate
```
Can you translate leftover entries like these?

efimov90 replied:
Sure, but not very fast. I will continue translating.

liach replied:
Will wait then

efimov90 replied:
Can you please explain what exact is Charge Trap? 715 tile.railcraft.charge.trap.name=Charge Trap

efimov90 committed:
second part of translate changes (2/3-4)

Update ru_RU translate

Left:

715
1015-1043 tips

1126, 1127 old tracks, does it need translate?

1161+

efimov90 committed:
last changes to make ru_RU like en_US last version

1191 and 1192 should not be translated - it is example for translators

Strap Iron - have no idea what it is
Charged trap - translated, but not sure it's correct

I will continue to translate.

liach commented on assets/railcraft/lang/ru_RU.lang:
```diff
@@ -1073,3 +1156,113 @@ tile.railcraft.wall.stone.brick.name=Стена из каменного кирп
+track_kit.railcraft.railcraft_whistle.tips=Проходящие локомотивы\n подуют в свисток\n§o§c- Необходим источник Redstone-сигнала -
+
+track_type.railcraft.railcraft_iron.name=Железный
+track_type.railcraft.railcraft_strap_iron.name=Strap Iron
```
This is old wooden track

liach commented on assets/railcraft/lang/ru_RU.lang:
```diff
@@ -1073,3 +1156,113 @@ tile.railcraft.wall.stone.brick.name=Стена из каменного кирп
+track_type.railcraft.railcraft_strap_iron.name=Strap Iron
+track_type.railcraft.railcraft_abandoned.name=Заброшенный
+track_type.railcraft.railcraft_reinforced.name=Усилинный
+track_type.railcraft.railcraft_high_speed.name=H. S.
```
This one means "High Speed" and below is "High Speed Electric"

efimov90 replied:
Ok, can i use full size words? Or better make like in en_us with 1st letters of words?

liach replied:
Either should be fine

efimov90 commented:
ok, so i translated traps right.

efimov90 committed:
Fixes to en_US like update

Strap iron and HS HSE rails translate update

liach commented:
Can I merge it now?

efimov90 commented:
Yes, i think this patch is ready now.
